kde-misc/zanshin/zanshin-0.4.0.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+CMAKE_MIN_VERSION="3.2.0"
+KDE_TEST="forceoptional"
+VIRTUALX_REQUIRED="test"
+inherit kde5
+
+DESCRIPTION="Getting things done application by KDE"
+HOMEPAGE="https://zanshin.kde.org/"
+[[ ${PV} == 9999 ]] || SRC_URI="http://files.kde.org/zanshin/${P}.tar.bz2"
+
+LICENSE="|| ( GPL-2 GPL-3 )"
+KEYWORDS="~amd64"
+IUSE=""
+
+# FIXME: bundles libkdepim
```

Comment:
yes, it looks very much out of date/sync compared to libkdepim.git so didn't try